pandas/core/indexes/interval.py

```diff
@@ -973,7 +973,7 @@ def interval_range(
         for numeric and 'D' for datetime-like.
     name : str, default None
         Name of the resulting IntervalIndex.
-    closed : {'left', 'right', 'both', 'neither'}, default 'right'
+    inclusive : {'left', 'right', 'both', 'neither'}, default 'right'
```

Comment:
you need to actually deprecate this, not just change it

Reply:
Have update it.